### PR TITLE
Adding hover to the list of opportunities with a pointer

### PR DESCRIPTION
**Pull request by yugmamoradia**
- Ticket: #151586176
screen shot 2017-10-06 at 11 53 46 am

### CODE REVIEW

**Comment by bbmoz** [comment]
@yugmamoradia update your branch with master!

File: client/src/components/ListOfProjects/ListOfProjects.css [hunk 1]
```diff
@@ -17,3 +17,8 @@
   background-color: darkgray;
   margin-bottom: 15px;
 }
+
+li:hover{
+  box-shadow: 0 6px 4px -2px #808080;
```
Comment (bbmoz) [inline client/src/components/ListOfProjects/ListOfProjects.css]: We should try to use color names wherever possible. I think that based off the design, a more accurate box-shadow would be:
```css
box-shadow: 0 3px 4px 1px gray;
```
https://projects.invisionapp.com/share/Z9DIWX2XM#/screens/257232552

File: .git-authors [hunk 2]
```diff
@@ -5,6 +5,9 @@ pairs:
   ym: Yugma Moradia; yugma93
   kj: Karthi Jagatheesan
   km: Klaus Mana
+  nw: Nathan Wang; nathan.wang2
+  sc: Serena Cao; wcao0630
+
```
Comment (bbmoz) [inline .git-authors]: This is weird. This is already part of master and not part of your changes. I would recommend you apply your changes on top of what's on master. I believe the only change you would make is in the `ListOfProjects.css` file.

Comment (yugmamoradia) [reply .git-authors]: I know. I made changes in `ListOfProjects.css` and nowhere else.

File: client/src/components/ListOfProjects/ListOfProjects.css (hunk 1 above)
Comment (yugmamoradia) [reply client/src/components/ListOfProjects/ListOfProjects.css]: Got it. Thanks

**Pull request closed without merging**